ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -171,6 +182,27 @@ public void shouldPrintTopic() throws Exception {
     assertThat(messages, hasSize(is(LIMIT)));
   }
 
+  @Test
+  public void shouldDeleteTopic() {
```

Comment:
Would be cool to add a test where the user doesn't have rights - and assert the error message is sensible.
Kind of outside the scope of this PR... but....